### ISSUE TITLE
Error auto updating to 1.1.3 on Mac

### ISSUE THREAD

**Issue by isoughtajam** [thenewinquiry/bailbloc#22]
Installed from source on Yosemite (10.10.5 maybe this is my fault) and on `npm start` got the following:
```
Checking for update
Generated new staging user ID: 245d4d5f-5b1a-5cae-8539-f274cf0976f3
Found version 1.1.3 (url: )
Downloading update from 
Error: Could not get code signature for running application
    at Server.<anonymous> (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/MacUpdater.ts:85:28)
    at Object.onceWrapper (events.js:293:19)
    at emitNone (events.js:86:13)
    at Server.emit (events.js:188:7)
    at emitListeningNT (net.js:1290:10)
    at _combinedTickCallback (internal/process/next_tick.js:77:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
Error: Error: Could not get code signature for running application
    at Server.<anonymous> (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/MacUpdater.ts:85:28)
    at Object.onceWrapper (events.js:293:19)
    at emitNone (events.js:86:13)
    at Server.emit (events.js:188:7)
    at emitListeningNT (net.js:1290:10)
    at _combinedTickCallback (internal/process/next_tick.js:77:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
[12:18:23.142] [warn] Error in auto-updater.
[12:18:23.146] [error] undefined
```
Seems to be running fine regardless, I'm guessing still 1.1.2 though.

**Comment by isoughtajam** [comment]
Update, app doesn't seem to be running. Also this:
```Checking for update
Found version 1.1.3 (url: )
Downloading update from 
Error: Could not get code signature for running application
    at Server.<anonymous> (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/MacUpdater.ts:85:28)
    at Object.onceWrapper (events.js:293:19)
    at emitNone (events.js:86:13)
    at Server.emit (events.js:188:7)
    at emitListeningNT (net.js:1290:10)
    at _combinedTickCallback (internal/process/next_tick.js:77:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
Error: Error: Could not get code signature for running application
    at Server.<anonymous> (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/MacUpdater.ts:85:28)
    at Object.onceWrapper (events.js:293:19)
    at emitNone (events.js:86:13)
    at Server.emit (events.js:188:7)
    at emitListeningNT (net.js:1290:10)
    at _combinedTickCallback (internal/process/next_tick.js:77:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
[12:48:23.161] [warn] Error in auto-updater.
[12:48:23.162] [error] undefined
Error: Error: Could not get code signature for running application
    at Server.<anonymous> (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/MacUpdater.ts:85:28)
    at Object.onceWrapper (events.js:293:19)
    at emitNone (events.js:86:13)
    at Server.emit (events.js:188:7)
    at emitListeningNT (net.js:1290:10)
    at _combinedTickCallback (internal/process/next_tick.js:77:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
From previous event:
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/MacUpdater.ts:44:12
From previous event:
    at MacUpdater.doDownloadUpdate (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/MacUpdater.js:127:11)
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:304:25
    at Generator.next (<anonymous>)
From previous event:
    at MacUpdater.downloadUpdate (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/AppUpdater.js:355:11)
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:280:49
From previous event:
    at MacUpdater.doCheckForUpdates (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/AppUpdater.js:326:11)
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:228:25
    at Generator.next (<anonymous>)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
From previous event:
    at MacUpdater._checkForUpdates (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/AppUpdater.js:280:11)
    at MacUpdater.checkForUpdates (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:177:35)
    at checkUpdates (/Users/gjoshi/dev/bailbloc/desktop/main.js:72:15)
    at App.app.on (/Users/gjoshi/dev/bailbloc/desktop/main.js:319:3)
    at emitTwo (events.js:111:20)
    at App.emit (events.js:194:7)
[12:48:23.164] [warn] Error in auto-updater.
[12:48:23.164] [error] Error: Could not get code signature for running application
    at Server.<anonymous> (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/MacUpdater.ts:85:28)
    at Object.onceWrapper (events.js:293:19)
    at emitNone (events.js:86:13)
    at Server.emit (events.js:188:7)
    at emitListeningNT (net.js:1290:10)
    at _combinedTickCallback (internal/process/next_tick.js:77:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
From previous event:
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/MacUpdater.ts:44:12
From previous event:
    at MacUpdater.doDownloadUpdate (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/MacUpdater.js:127:11)
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:304:25
    at Generator.next (<anonymous>)
From previous event:
    at MacUpdater.downloadUpdate (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/AppUpdater.js:355:11)
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:280:49
From previous event:
    at MacUpdater.doCheckForUpdates (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/AppUpdater.js:326:11)
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:228:25
    at Generator.next (<anonymous>)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
From previous event:
    at MacUpdater._checkForUpdates (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/AppUpdater.js:280:11)
    at MacUpdater.checkForUpdates (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:177:35)
    at checkUpdates (/Users/gjoshi/dev/bailbloc/desktop/main.js:72:15)
    at App.app.on (/Users/gjoshi/dev/bailbloc/desktop/main.js:319:3)
    at emitTwo (events.js:111:20)
    at App.emit (events.js:194:7)
Unhandled rejection Error: Could not get code signature for running application
    at Server.<anonymous> (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/MacUpdater.ts:85:28)
    at Object.onceWrapper (events.js:293:19)
    at emitNone (events.js:86:13)
    at Server.emit (events.js:188:7)
    at emitListeningNT (net.js:1290:10)
    at _combinedTickCallback (internal/process/next_tick.js:77:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
From previous event:
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/MacUpdater.ts:44:12
From previous event:
    at MacUpdater.doDownloadUpdate (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/MacUpdater.js:127:11)
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:304:25
    at Generator.next (<anonymous>)
From previous event:
    at MacUpdater.downloadUpdate (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/AppUpdater.js:355:11)
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:280:49
From previous event:
    at MacUpdater.doCheckForUpdates (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/AppUpdater.js:326:11)
    at /Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:228:25
    at Generator.next (<anonymous>)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
From previous event:
    at MacUpdater._checkForUpdates (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/out/AppUpdater.js:280:11)
    at MacUpdater.checkForUpdates (/Users/gjoshi/dev/bailbloc/desktop/node_modules/electron-updater/src/AppUpdater.ts:177:35)
    at checkUpdates (/Users/gjoshi/dev/bailbloc/desktop/main.js:72:15)
    at App.app.on (/Users/gjoshi/dev/bailbloc/desktop/main.js:319:3)
    at emitTwo (events.js:111:20)
    at App.emit (events.js:194:7)```

Digging into it in a bit.

**Comment by antiboredom** [comment]
@isoughtajam actually can you do a git pull and try again? Think I may have resolved it...

**Comment by isoughtajam** [comment]
Looks fixed, I'm running 1.1.3! And I can tell it's running because of this?
```
69334 ttys004   48:23.91 /.../bailbloc/desktop/miner_binaries/bailbloc_worker --url=mine.xmrpool.net:5555 --user=442uGwAdS8c3mS46h6b7KMPQiJcdqmLjjbuetp
```

**Comment by antiboredom** [comment]
Yeah, looks like it's working. The bailblock_worker is the actual miner.
